### PR TITLE
Adds Kelly Madison Media xPath CDP scraper

### PR DESCRIPTION
**Pull request by halorrr**

### CODE REVIEW

File: scrapers/KellyMadisonMedia.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      Title:
+        selector: $script
+        replace:
+          - regex: .+(?:chromecast.+\stitle:\s')([^']+).+
+            with: $1
+          - regex: '&#039;'
+            with: "'"
+      Details: $summary/div/div[2]/p/text()
+      Date:
+        selector: //div[@class="card episode-summary mb-4"]/div/div[1]/h4[contains(.,'Published:')]/text()[2]
```
Comment (bnkai) [inline scrapers/KellyMadisonMedia.yml:28]: the `$summary` can be used here and also `div[1]` removed and replaced 
```yaml
      Date:
        selector: $summary//div/h4[contains(.,'Published:')]/text()[2]
```

File: scrapers/KellyMadisonMedia.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      $script: //script[contains(text(),"chromecast")]/text()
+    scene:
+      Title:
+        selector: $script
+        replace:
+          - regex: .+(?:chromecast.+\stitle:\s')([^']+).+
+            with: $1
+          - regex: '&#039;'
+            with: "'"
+      Details: $summary/div/div[2]/p/text()
```
Comment (bnkai) [inline scrapers/KellyMadisonMedia.yml:26]:
```yaml
     Details: $summary//div[contains(h4,'Episode Summary')]/p/text()
```
is probably safer

File: scrapers/KellyMadisonMedia.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      Date:
+        selector: //div[@class="card episode-summary mb-4"]/div/div[1]/h4[contains(.,'Published:')]/text()[2]
+        replace:
+          - regex: Published:\s
+            with:
+        parseDate: 2006-01-02
+      Performers:
+        Name: $summary/div/div[1]//a/text()
```
Comment (bnkai) [inline scrapers/KellyMadisonMedia.yml:34]:
```yaml
      Performers:
        Name: $summary/div//div/h4[contains(.,'Starring')]/a/text()
```
probably safer

**Comment by halorrr** [comment]
Updated!

**Pull request merged**